<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Page</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="UserLoginlabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passLinedit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="passLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDlineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="role_combobox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Admin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Driver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Passenger</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="role_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Role:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Loginbtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newUserbtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New User?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
